### DISCUSSION TITLE
Monthly ESMValTool meeting February 2022

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#9] [Meetings]
The next monthly ESMValTool meeting will be in February. The result of the poll is that it will be on

Monday, February 7 at 12:00 CET (11:00 GMT).

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

@ESMValGroup/esmvaltool-developmentteam

Format and agenda

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we can decide to talk about specific topics with the entire group or make smaller breakout groups to continue the discussion there.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the GitHub issue on the topic in a comment below. That will make it easy to find for other people in the meeting and we can already start discussing there before the meeting. Just create a new GitHub issue about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the November meeting can be found here: https://github.com/ESMValGroup/Community/discussions/5#discussioncomment-1737536

**Comment by bouweandela** [comment]
@ESMValGroup/esmvaltool-developmentteam I intend to pick the date for the meeting tonight, so please fill the doodle today

**Comment by sloosvel** [comment]
Hi, is this meeting happening today at the end?

**Comment by schlunma** [comment]


**Reply by bouweandela** [reply]
We're all in this meeting

**Comment by bouweandela** [comment]
Personal updates from the update round:

- @schlunma @axel-lauer @remi-kazeroni Are the release managers for the [upcoming v2.5 release](https://docs.esmvaltool.org/en/latest/community/release_strategy.html#upcoming-releases) https://github.com/ESMValGroup/ESMValCore/discussions/1450 https://github.com/ESMValGroup/ESMValCore/discussions/1446 https://github.com/ESMValGroup/ESMValTool/discussions/2495. They will be supported by the previous release manager @zklaus.
- @hb326 @remi-kazeroni Would like to talk about the next workshop
- @ledm See [the 11 slides that ...](https://www.carbonbrief.org/revealed-the-11-slides-that-finally-convinced-boris-johnson-about-global-warming) (slide 7) for some awesome ESMValTool policy impact. Working on global warming thresholds with @rswamina. Now that we have a feature for downloading data that is not available locally, how do we find out what data is available on ESGF that we could download?
- @rswamina Working on teaching a tutorial with @remi-kazeroni and @zklaus, updating actual tutorial with @SarahAlidoost and @LisaBock
- @alistairsellar Please comment on the backward compatibility discussion at #7 
- @katjaweigel Worked on CMUG pull requests
- @zklaus Working on regridding with better support for non-regular grids through iris-esmf in https://github.com/ESMValGroup/ESMValCore/pull/1448. Worked on improving GitHub actions together with @valeriupredoi
- @ChrisJones-MOHC would like to talk about fx files issue
- @ehogan ESMValTool community environment set up at the Met Office, `module load esmvaltool` now available at the Met Office, observation data still an issue. @bouweandela recommended mentioning this installation method in the [installation documentation](https://docs.esmvaltool.org/en/latest/quickstart/installation.html#pre-installed-versions-on-hpc-clusters) alongside instructions for Mistral and Jasmin. @valeriupredoi suggested working together with himself and @remi-kazeroni on the topic of obtaining observational datasets.
- @sloosvel Working on finalizing pull request with a recipe for monitoring on-going model runs in https://github.com/ESMValGroup/ESMValTool/pull/2184. @valeriupredoi offers to help
- @bsolino May be working more on the tool soon
- @valeriupredoi Helping out various people
- @bettina-gier Writing papers using tool

**Comment by bouweandela** [comment]
*fx files discussion*

@ChrisJones-MOHC reported a discontinuity in his timeseries because the land fraction for NorESM is different in the historical vs future runs. He asked if it was possible to select which fx file to use for which model. @bouweandela said that this is at the moment only possible per preprocessor, by overriding the facets (e.g. `exp: historical`) from the dataset definition there. @bouweandela said that he hopes to have time to improve this, see https://github.com/ESMValGroup/ESMValCore/issues/1138.

@ChrisJones-MOHC  Will create an issue to discuss this in further detail.

**Reply by bouweandela** [reply]
The issue is here https://github.com/ESMValGroup/ESMValCore/issues/1474. @rswamina I believe you were also interested in the topic?

**Comment by bouweandela** [comment]
*upcoming release*
@schlunma  reported that there are 2 open pull requests remaining in ESMValCore and then release candidate will be created. @remi-kazeroni will run all recipes and then @schlunma will reach out to all authors for reviewing the recipe results. Help with reviewing would be appreciated later this week or next week.

**Comment by bouweandela** [comment]
*upcoming workshop*

It looks like there is a preference for in person, but with hybrid option for people who are unable to travel. If anyone is interested in joining the organization or even taking over, please contact @remi-kazeroni or @axel-lauer. Many events are planned in May or June because people hope that COVID restrictions will be lifted by then. @katjaweigel remarked that choosing this timeslot will create competition with other events, as it is not possible to go everywhere so a hybrid in-person/online option would definitely be appreciated.

**Comment by bouweandela** [comment]
*user engagement team*

@rswamina announced that the next @ESMValGroup/userengagementteam meeting will take place Mon 21 Feb 2022 15:00 – 16:00 (CET). Please contact the team [here](https://github.com/ESMValGroup/Community/blob/main/User%20Engagement%20Lead/README.md) if you would like to join.

**Reply by bouweandela** [reply]
Here is the relevant discussion: #14

**Comment by bouweandela** [comment]
*automatic download and CMORization of observational and re-analysis datasets*

@remi-kazeroni will take over https://github.com/ESMValGroup/ESMValTool/pull/1657 from @jvegreg and get it included in the upcoming v2.5 release. There was some discussion on whether this could be included in a minor release (see #7), but people felt that this could not wait because new CMORizers are added all the time, making the pull request out of date. It was felt that this will be a big improvement w.r.t. the current situation. It seems unlikely that people will use the current set of CMORizers without editing the source code either, so postponing to just after the release would not help either. Therefore the release managers decided that this will be included in v2.5.

**Comment by bouweandela** [comment]
At the meeting I promised I would share a script I wrote some time ago for finding out what data is available on ESGF and putting it in a recipe. I just shared it here: https://github.com/ESMValGroup/ESMValCore/issues/1484.
